client/drum-pad.html

```diff
@@ -1,29 +1,38 @@
-<div ng-controller="drumPad">
+<div class="app-container" ng-controller="drumPad">
```

Comment:
Makes sense. This was the beginnings of a next step. I'll remove it for this pull request.